polyglot/piranha/test-resources/java/feature_flag_system_2/control/expected/XPMethodChainCases.java

```diff
@@ -25,7 +25,7 @@ public void testDontMatchNonInstanceNested() {
 
   public static void foobar(Parameter cp) {
     SomeParam sp = SomeParam.create(cp);
-    // Matches API
+
```

Comment:
Shouldn't we be cleaning up these empty lines as well? We do elsewhere, see [XPFlagCleanerInterfaceMethod.java](https://github.com/uber/piranha/pull/237/files#diff-ce6a072c15af6643a7138283467c1ba0e73fcd45326dd7dc24c4211a15cff607) right below.